docs/run/agent/report-hub.md

```diff
@@ -35,3 +35,10 @@ Some common options are:
 These options will be reflected in the `helm install` command you'll see when you click the `Install` icon.
 Be sure to run the new command after making any changes here.
 
+## Report History
+The `History` tab will show you a list of recent reports for each report type.
+
+<img :src="$withBase('')" alt="report history">
```

Comment:
I haven't seen this approach before. Does this show an image on deployment? The rich diff is showing a missing image.

Reply:
🤦 forgot to add the image!